.gitignore

```diff
@@ -16,3 +16,4 @@
 coverage.txt
 codegpt
 dist
+.idea/
```

Comment:
Error generating comment for file .gitignore: Post "https://api.openai.com/v1/chat/completions": EOF